lib/autoload/polyglot/language/python.rb

```diff
@@ -0,0 +1,25 @@
+class Polyglot::Language::Python < Polyglot::Language
+  class Python2Point7 < Polyglot::Language::Python
+    class << self
+      def instance
+        root
```

Comment:
Not too good, because there's Python 2.7 and Python 3.4, which are not base, Python is. But Python 2.7 and Python 3.4 are system-defined languages